### PR TITLE
[Task 2] Integrate with AWS S3 and CloudFront via npm scripts

### PR DESCRIPTION
**Pull request by demakoff**
**1**. https://github.com/rolling-scopes-school/nodejs-aws-tasks/blob/main/task2-serve-spa-aws/task.md
**2**. 
image
**3**. 
S3 website: http://dide-my-second-app.s3-website-eu-west-1.amazonaws.com/ (403 Forbidden)
Cloudfront url: https://du9txjlt4cpdh.cloudfront.net/
**4**. 21.10.2020 / 24.10.2020
**5.** 5/5

### CODE REVIEW

**Comment by KarneyenkaDzmitry** [comment]
Hello Dmitry,

Could you also share the links to the first (manual) - implementation of the task?

**Comment by tensegrity666** [comment]
Yep, without it, according to the conditions, only 4/5

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,9 @@ yarn-error.log*
 
 /.idea
 .vscode
+
+jspm_packages
+
+# Serverless directories
+.serverless
```
Comment (ArthurKa) [inline .gitignore]: Duplicate of line 13.

**Review by ArthurKa — COMMENTED** [review]
Why `package-lock.json` and `yarn.lock` are provided at the same time? Is there any specific reason for that?

File: .gitignore (hunk 1 above)
Comment (demakoff) [reply .gitignore]: yep, thanks

Comment (ArthurKa) [reply .gitignore]: You are welcome)

**Comment by demakoff** [comment]
> Hello Dmitry,
> 
> Could you also share the links to the first (manual) - implementation of the task?

Could you please help me find why should I provide it to get highest score?

image

**Comment by demakoff** [comment]
> Why `package-lock.json` and `yarn.lock` are provided at the same time? Is there any specific reason for that?

Honestly there is no specific reason, just used both npm and yarn, will leave one, thanks

**Comment by demakoff** [comment]
BTW, update from our Lecturer:
image

File: .gitignore (hunk 1 above)
Comment (demakoff) [reply .gitignore]: thanks, updated

**Review by KarneyenkaDzmitry — APPROVED** [review]

**Pull request merged**